nautobot/docs/installation/nautobot.md

```diff
@@ -59,30 +49,42 @@ In the following steps, we will have you create the virtualenv within the `NAUTO
 
 ### Create the Virtual Environment
 
-We're going to create the virtualenv in our `NAUTOBOT_ROOT` as the `nautobot` user to populate the `/opt/nautobot` directory with a self-contained Python environment including a `bin` directory for scripts and a `lib` directory for Python libaries.
+As root, we're going to create the virtualenv in our `NAUTOBOT_ROOT` as the `nautobot` user to populate the `/opt/nautobot` directory with a self-contained Python environment including a `bin` directory for scripts and a `lib` directory for Python libaries.
 
 ```no-highlight
-$ python3 -m venv /opt/nautobot
+$ sudo -u nautobot python3 -m venv /opt/nautobot
 ```
 
 ### Update the Nautobot `.bashrc`
 
 So what about the `NAUTOBOT_ROOT`? We've referenced this environment variable several times. Here is where it finally gets set.
 
-We need to set the `NAUTOBOT_ROOT` environment variable for the `nautobot` user and make sure that it always set without having to do it manually. Add this to `~/.bashrc` for `nautobot` so that anytime you become `nautobot`, your `NAUTOBOT_ROOT` will be set automatically.
+We need to set the `NAUTOBOT_ROOT` environment variable for the `nautobot` user and make sure that it always set without having to do it manually.
+
+Run this command to update `~/.bashrc` for `nautobot` so that anytime you become `nautobot`, your `NAUTOBOT_ROOT` will be set automatically.
 
 ```no-highlight
-$ echo "export NAUTOBOT_ROOT=/opt/nautobot" >> ~/.bashrc
+$ echo "export NAUTOBOT_ROOT=/opt/nautobot" | sudo tee -a ~nautobot/.bashrc
 ```
 
-Next, reload the `.bashrc` file so that your `NAUTOBOT_ROOT` gets set and then echo its value to verify it worked:
+## Sudo to nautobot
```

Comment:
I think this needs to be left near the top or we need to change the previous prompts to `#` to avoid confusion.  It might even be worth explicitly adding the user to each instruction like `root@nautobot #` and `nautobot@nautobot $`.  Line 52 says "As root", but the code example makes it ambiguous IMO.

Reply:
Yeah, you're raising a good point, but I don't really want to recommend that people actually login or sudo _to_ root, but rather to run things _with root permissions_. 

Really all of our "as root" instructions really mean "use sudo", which allows one to run commands _with root permissions_. I am Groot?

Do you think that clarification would help?

Documentation is hard, y'all!

Reply:
After looking back through the install docs, we don't use `#` anywhere, so ignore me.  You are correct, "with privileges" is pretty clear if I'd pay closer attention.  Looks good to me!